src/wp-includes/functions.php

```diff
@@ -3672,7 +3672,7 @@ function get_allowed_mime_types( $user = null ) {
  */
 function wp_nonce_ays( $action ) {
 	// Default title and response code.
-	$title         = __( 'Something went wrong.' );
+	$title         = __( 'An unexpected error occurred. Please try again later.' );
```

Comment:
Links are provided to try again, so this can either stay as "something went wrong" or "an error occurred".